packages/tables/src/Filters/SelectFilter.php

```diff
@@ -39,7 +39,7 @@ protected function setUp(): void
                 __('filament-tables::table.filters.select.placeholder'),
         );
 
-        $this->indicateUsing(function (SelectFilter $filter, array $state): array {
+        $this->indicators(function (SelectFilter $filter, array $state): array {
```

Comment:
@danharrin you mentioned that we can indicate on selected items and remove them individually, do you think we should make that the default here to make that functionality more discoverable?

Reply:
At the moment, each default filter only has one indicator for simplicity and consistency

Reply:
Yeah I think it makes sense to keep it simple by default. Especially since it's relatively easy to set up separate indicators for each value.